chalice/deploy/deployer.py

```diff
@@ -198,19 +198,20 @@ def deploy(self, config, chalice_stage_name=DEFAULT_STAGE_NAME):
         existing_resources = config.deployed_resources(chalice_stage_name)
         deployed_values = self._lambda_deploy.deploy(
             config, existing_resources, chalice_stage_name)
+        deployed_values.update({
```

Comment:
Not needed for a specific feature, just general cleanup.  Fill in all the values we now up front, and only update the deployed values with whatever the current "deployer" creates.